sign_language_datasets/datasets/dgs_corpus/dgs_corpus.py

```diff
@@ -10,10 +13,11 @@
 import tensorflow_datasets as tfds
 
 from os import path
-from typing import Dict, Any, Set, Optional, List
+from typing import Dict, Any, Set, Optional, List, Literal
```

Comment:
I currently get on Colab:

`ImportError: cannot import name 'Literal' from 'typing' (/usr/lib/python3.7/typing.py)`

(Because it's a Python>=3.8 feature, https://stackoverflow.com/questions/61206437/importerror-cannot-import-name-literal-from-typing)

could we omit this to support Colab?